clonelists/Nintendo - GameCube.json

```diff
@@ -450,6 +453,8 @@
 		],
 		"Nickelodeon SpongeBob SquarePants - The Movie": [
 			["Nickelodeon SpongeBob Schwammkopf - Der Film", 1]
+			["2 Games in 1 - Bob L'eponge - Le Film + Tak 2 - Le Sceptre des Reves (France) (Disc 1) (Bob L'eponge - Le Film)", 1]
+			["2 Games in 1 - Nickelodeon SpongeBob Schwammkopf - Der Film + Nickelodeon SpongeBob Schwammkopf - Schlacht um Bikini Bottom (Germany) (Disc 1)", 1]
```

Comment:
For this to work, you need use [short names](https://github.com/unexpectedpanda/retool/wiki/Terminology#short-names). For the titles you've added, this means removing the countries.

Reply:
Got it!

Reply:
Feel free to update the commit, alternatively I can roll the changes separately into the next update :)

Reply:
Hi! I'm a little busy and I'm waiting for a massive dl to finish so I can test and append fixes.

Reply:
No worries -- I'll close out this PR and integrate the corrections directly :)